packages/p2p/src/components/buy-sell/nickname-form.jsx

```diff
@@ -92,6 +92,18 @@ const NicknameForm = ({ handleClose, setNickname, setChatInfo }) => {
                                         )}
                                     </Field>
                                 </div>
+                                <div className='buy-sell__popup-content--list'>
+                                    <div>
+                                        {localize(
+                                            'Must be 2–24 characters and can contain letters, numbers, and special characters .- _ @'
+                                        )}
+                                    </div>
+                                    <div>{localize('Cannot repeat a character more than 5 times')}</div>
+                                    <div>{localize('Cannot start, end with, or repeat special characters')}</div>
```

Comment:
- Why not use `ul` elements? 🤔 
- All sentences lack a period (`.`) at the end as per the design.

Reply:
> Why not use ul elements? 
hmm, idk. Maybe i was drunk 😂